js/app/interfaces/card.js

```diff
@@ -3,23 +3,32 @@
 /* exported Card, MinSize, MaxSize */
 
 const EosKnowledgePrivate = imports.gi.EosKnowledgePrivate;
+const Format = imports.format;
 const GdkPixbuf = imports.gi.GdkPixbuf;
+const Gettext = imports.gettext;
 const Gio = imports.gi.Gio;
 const GLib = imports.gi.GLib;
 const GObject = imports.gi.GObject;
 const Gtk = imports.gi.Gtk;
 const Lang = imports.lang;
 const Mainloop = imports.mainloop;
 
+const Config = imports.app.config;
 const ContentObjectModel = imports.search.contentObjectModel;
+const Engine = imports.search.engine;
 const FormattableLabel = imports.app.widgets.formattableLabel;
 const ImageCoverFrame = imports.app.widgets.imageCoverFrame;
 const Module = imports.app.interfaces.module;
+const QueryObject = imports.search.queryObject;
 const SearchUtils = imports.search.utils;
 const SetMap = imports.app.setMap;
+const SetObjectModel = imports.search.setObjectModel;
 const SpaceContainer = imports.app.widgets.spaceContainer;
 const Utils = imports.app.utils;
 
+String.prototype.format = Format.format;
+let _ = Gettext.dngettext.bind(null, Config.GETTEXT_PACKAGE);
```

Comment:
Call this function `ngettext`, otherwise it will be scanned incorrectly when generating the pot files